### PR TITLE
codegen: do not generate code for dispatch_namespaced_definitions

### PR DESCRIPTION
**Pull request by guoyejun**

### CODE REVIEW

**Comment by guoyejun** [comment]
hi, I see three failures:
2 new failures recognized by patterns
1 fixed upstream failure:

I checked the build log and did not find hint, I just wonder the first 2 failures are the same as the third one, they are all not introduced by this change, how could I verify it? thanks.

File: tools/codegen/gen_backend_stubs.py [hunk 1]
```diff
@@ -233,16 +233,7 @@ def make_file_manager(install_dir: str) -> FileManager:
...
-                    dest.RegisterDispatchKey(
-                        backend_indices[dispatch_key],
-                        Target.NAMESPACED_DEFINITION,
-                        selector,
-                        rocm=False,
-                        cpp_namespace=cpp_namespace,
-                        class_method_name=f'{backend_dispatch_key}NativeFunctions'),
-                    grouped_native_functions
-                )),
+                'dispatch_namespaced_definitions': '',
```
Comment (bdhirsh) [inline tools/codegen/gen_backend_stubs.py:236]: can you add a light comment - external backends don't need namespaced definitions now, but we can always add them back later.

**Review by bdhirsh — APPROVED** [review]
Ok! We can always add this back later if there's a need for it for external backends.

**Comment by bdhirsh** [comment]
I see an error
```
Cannot find source file:

    /var/lib/jenkins/workspace/aten/src/THC/THCStorage.cpp
```

Which looks very unrelated to this PR

**Comment by bdhirsh** [comment]
can you rebase this? I think there are some merge conflicts.

**Comment by guoyejun** [comment]
done, thanks.

**Comment by guoyejun** [comment]
hi, any comment? thanks.

**Comment by bdhirsh** [comment]
Sorry, this slipped past me. Trying to re-land!

**Comment by guoyejun** [comment]
where to add such label in the PR? thanks.

**Comment by bdhirsh** [comment]
That's for our release process, (although cc'ing the OSS owner of the PR is a little confusing). I just added the labels.

**Comment by guoyejun** [comment]
thanks

**Pull request closed without merging**